backend/TreeOfAKind.API/Trees/RemoveTreePhotoRequest.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
```

Comment:
You could do this by yourself on backend when photo link is updated with null or empty string.

Reply:
Idk this gives clearer intention it is more readable.
It will be easier to spot in logs if mobile/web is sending a proper file data